File: Java/UberVan.java
import java.util.Map;
import java.util.ArrayList;

public class UberVan extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license,String driver){
            super(license, driver);
    }

    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }

}

File: Python/UberVan.py
from car import Car 

class UberVan(Car):
    typerCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typerCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial